### PR TITLE
Move generic k8s tools to k8s library from k8sbasetest

### PR DESCRIPTION
**Pull request by ilausuch**
In order to use these tools without having a test with k8sbasetest as base. The best option is to move these tools to k8s library

- Related ticket: https://progress.opensuse.org/issues/117004
- Verification run: https://openqa.suse.de/tests/9646619

### CODE REVIEW

File: lib/containers/k8s.pm [hunk 1]
```diff
@@ -92,4 +112,47 @@ sub install_oc {
...
+    upload_logs($path, failok => 1);
+
+    assert_script_run("kubectl apply -f $path");
+}
+
+=head2 find_pods
+Find pods using kubectl queries
+=cut
+
+sub wait_for_job_complete {
```
Comment (mloviska) [inline lib/containers/k8s.pm:135]: I guess, it would be less ambiguous to express that the function waits for a `kube` job/pod to finished

Comment (ilausuch) [reply lib/containers/k8s.pm:135]: good point, but it is inside k8s lib, is necessary to specify? wait_for_k8s_job_complete, for instance?

Comment (mloviska) [reply lib/containers/k8s.pm:135]: I think that it would be handy, mainly for people that are often `greping` test repo for function names as me. :D

Comment (ilausuch) [reply lib/containers/k8s.pm:135]: Fixed then

**Pull request merged**